### PR TITLE
updated ina219 to work with nano

### PR DESCRIPTION
**Pull request by rschiefer**
## Description

Migrated Ina219 device to nF.

## Motivation and Context

- Fixes https://github.com/nanoframework/Home/issues/856

## How Has This Been Tested?

I validated sample runs successfully with KeeYees ESP-WROOM-32 CP2102 and HiLetgo INA219 (https://www.amazon.com/dp/B07VL8NY32?psc=1&ref=ppx_yo2_dt_b_product_details)

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (work on Unit Tests, has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: devices/Ina219/README.md [hunk 1]
```diff
@@ -53,5 +58,3 @@ at 5mV. Given this we can use a shunt voltage range of +/- 40mV
 * The maximum possible current would then be 40mV / 0.1 = 400mA
 * With a 400mA maximum current and a range of the ADC of 15bits then the LSB of the current would be 400mA/32767 = 12.2207 microamps. We will chose 12.2uA as a round number.
 * From the [INA219 Datasheet](http://www.ti.com/lit/ds/symlink/ina219.pdf) the calibration register should be set at 0.04096/(currentLSB * shunt resistance) = 33574 = 0x8326
-
-![circuit](Ina219.Sample_bb.png)
```
Comment (Ellerbach) [inline devices/Ina219/README.md:57]: I saw you added a fritzing diagram, I guess it's to place here, right?

**Review by Ellerbach — APPROVED** [review]
Another great migration @rschiefer !!! Thanks a lot, looks great!

File: devices/Ina219/README.md (hunk 1 above)
Comment (josesimoes) [reply devices/Ina219/README.md:57]: @rschiefer care to take care of this tiny nit so this can be merged? 😉

Comment (rschiefer) [reply devices/Ina219/README.md:57]: Sorry, I missed this.  Yes, I will fix it

**Comment by josesimoes** [comment]
@rschiefer happy new year! 😃 
Can you please let us know if you're planning to wrap up this one or not.

**Comment by josesimoes** [comment]
OK. Please just rebase so this one can be merged.

**Comment by rschiefer** [comment]
working to fix the build

**Comment by Ellerbach** [comment]
@rschiefer thanks for the efforts, I fixed the latest issues ;-)

**Comment by rschiefer** [comment]
Thanks @Ellerbach!

**Pull request merged**